airbyte-integrations/connectors/source-mssql/Dockerfile

```diff
@@ -24,5 +24,5 @@ ENV APPLICATION source-mssql
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=1.1.1
+LABEL io.airbyte.version=2.0.0
```

Comment:
Do you also want to do this for mssql-strict-encrypt?

Reply:
Good call. Will add that